chia/cmds/chia.py

```diff
@@ -132,7 +132,10 @@ def run_daemon_cmd(ctx: click.Context, wait_for_unlock: bool) -> None:
 
 
 def main() -> None:
-    cli()  # pylint: disable=no-value-for-parameter
+    try:
+        cli()  # pylint: disable=no-value-for-parameter
+    except CliRpcConnectionError:  # this happens when we cant connect to a client and is only comes from the cli code
+        exit(1)
```

Comment:
we don't have any error message here?

Reply:
error message is already printed out by the generator.